examples/trivial.go

```diff
@@ -36,18 +37,29 @@ func subRoutine(ctx context.Context) {
 	defer subSpan.Finish()
 }
 
+type LoggingRecorder struct {
+	r lightstep.SpanRecorder
+}
+
+func (r *LoggingRecorder) RecordSpan(span lightstep.RawSpan) {
+	logger.Printf("span traceID: %v spanID: %v parentID: %v Operation: %v \n", span.Context.TraceID, span.Context.SpanID, span.ParentSpanID, span.Operation)
+}
+
 func main() {
 	flag.Parse()
 	if len(*accessToken) == 0 {
 		fmt.Println("You must specify --access_token")
 		os.Exit(1)
 	}
 
+	loggableRecorder := &LoggingRecorder{}
```

Comment:
Is there a reason to have this variable? Can we just pass `&LoggingRecorder{}` into `lightstep.Options` below?